x/hard/keeper/borrow_test.go

```diff
@@ -561,4 +751,32 @@ func (suite *KeeperTestSuite) TestValidateBorrow() {
 		sdk.NewCoins(sdk.NewCoin("ukava", availableToBorrow.AmountOf("ukava"))),
 	)
 	suite.Require().NoError(err)
+
+	// now that it's all that you can borrow, we shouldn't be able to borrow anything
+	err = suite.keeper.Borrow(
+		suite.ctx,
+		borrower,
+		sdk.NewCoins(sdk.NewCoin("ukava", sdkmath.NewInt(2*USDX_CF))),
+	)
+	suite.Require().Error(err)
```

Comment:
Would be good practice to check the exact error values as much as possible instead of just being any error

Same for the other ones below